Importing..
Using parallel_transport = True
Using RPA
FRIIRGcat_final
Starting 1000 Monte Carlo simulations for n = 0 to n = 800
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:156: RuntimeWarning: invalid value encountered in arccos
  xi_1 = np.arccos( (np.sin(DEC_t)*np.sin(RA_t - RA_s)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:159: RuntimeWarning: invalid value encountered in arccos
  xi_2 = np.arccos( (-1* np.sin(DEC_s)*np.sin(RA_s - RA_t)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:156: RuntimeWarning: invalid value encountered in arccos
  xi_1 = np.arccos( (np.sin(DEC_t)*np.sin(RA_t - RA_s)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:159: RuntimeWarning: invalid value encountered in arccos
  xi_2 = np.arccos( (-1* np.sin(DEC_s)*np.sin(RA_s - RA_t)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:156: RuntimeWarning: invalid value encountered in arccos
  xi_1 = np.arccos( (np.sin(DEC_t)*np.sin(RA_t - RA_s)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:159: RuntimeWarning: invalid value encountered in arccos
  xi_2 = np.arccos( (-1* np.sin(DEC_s)*np.sin(RA_s - RA_t)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:156: RuntimeWarning: invalid value encountered in arccos
  xi_1 = np.arccos( (np.sin(DEC_t)*np.sin(RA_t - RA_s)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
/home/data0/lbq/RGC-Mask-Transfiner/tools/inference/FIRST/properties_analysis/Position_angle/utils.py:159: RuntimeWarning: invalid value encountered in arccos
  xi_2 = np.arccos( (-1* np.sin(DEC_s)*np.sin(RA_s - RA_t)) / (np.sqrt(1-(np.dot(u_rt,u_rs))**2 )) )[0]
Using 4 cores
